src/main/fc/fc_msp.c

```diff
@@ -288,20 +278,19 @@ static const box_t *findBoxByPermenantId(uint8_t permenantId)
 
 static void serializeBoxNamesReply(sbuf_t *dst)
 {
-    int activeBoxId, flag = 1, count = 0, len;
+    int flag = 1, count = 0;
 
 reset:
     // in first run of the loop, we grab total size of junk to be sent
     // then come back and actually send it
     for (int i = 0; i < activeBoxIdCount; i++) {
-        activeBoxId = activeBoxIds[i];
-
+        const int activeBoxId = activeBoxIds[i];
         const box_t *box = findBoxByActiveBoxId(activeBoxId);
         if (!box) {
             continue;
         }
 
-        len = strlen(box->boxName);
+        const int len = strlen(box->boxName);
```

Comment:
This is unchanged by me. Yes, it should probably be corrected, but in another PR. This PR contains just some minimal changes to clean things up and save some ROM.